EIPS/eip-4844.md

```diff
@@ -344,11 +344,12 @@ def validate_blob_transaction_wrapper(wrapper: BlobTransactionNetworkWrapper):
     versioned_hashes = wrapper.tx.message.blob_versioned_hashes
     commitments = wrapper.blob_kzgs
     blobs = wrapper.blobs
+    proofs = wrapper.blob_proofs
     # note: assert blobs are not malformatted
     assert len(versioned_hashes) == len(commitments) == len(blobs)
```

Comment:
```suggestion
    assert len(versioned_hashes) == len(commitments) == len(blobs) == len(proofs)
```

Reply:
(`verify_blob_kzg_proof_batch` will also perform this check so it's not essential here, but nice to make it very explicit. If we say that we do the verification in that function, then `assert len(versioned_hashes) == len(commitments)` would be enough because it covers both `blobs` and `proofs`)